src/components/bordered_pulldown.tsx

```diff
@@ -48,9 +56,20 @@ export class BorderedPulldown extends React.Component<Props, State> {
     })
 
     const displayValue = this.state.selected && this.state.selected.name || defaultValue
+    let pulldownStyles = {}
+
+    if (this.state.isHovered) {
+      pulldownStyles = {
+        display: "block",
+      }
+    }
```

Comment:
Switches the dropdown reveal from being CSS to JS/State based. The reason is so we can force an "unhover" when a selection is made.